market/service_proposal_test.go

```diff
@@ -59,7 +60,8 @@ func Test_ServiceProposal_Serialize(t *testing.T) {
       "quality": {
         "quality": 2.0,
         "latency": 5,
-        "bandwidth": 100
+        "bandwidth": 100,
+		"uptime": 20
```

Comment:
broken alignment?

Reply:
yes, it look like it broke a linter. will fix now